src/pages/room/[id]/index.vue

```diff
@@ -4,26 +4,26 @@
       <div class="row justify-between items-center q-mb-md">
         <div>
           <p class="text-h5 text-weight-bold q-mb-xs">
-            ディジェクマクン
+            {{ roomDetails.name }}
           </p>
           <p class="q-mb-none">
-            Life is Tech! Summer Camp 2022 関西大学D日程
+            {{ roomDetails.description }}
           </p>
         </div>
-        <q-btn color="primary" label="ルームを編集する" />
+        <q-btn :to="`/room/${roomDetails.displayId}/edit`" color="primary" label="ルームを編集する" />
       </div>
-      <div class="row q-mb-xs">
-        <v-icon icon="mdi-link" class="q-mr-xs" />
+      <div class="row items-center q-mb-xs">
+        <q-icon name="link" class="q-mr-xs" />
         <p class="q-mb-none">
           リクエストURL:
-          <a :href="requestUrl" target="_blank">{{ requestUrl }}</a>
+          <a :href="roomDetails.requestUrl" target="_blank">{{ roomDetails.requestUrl }}</a>
         </p>
       </div>
-      <div class="row">
-        <v-icon icon="mdi-spotify" class="q-mr-xs" />
+      <div class="row items-center">
+        <q-icon :name="roomDetails.provider=='spotify'?'fa-brands fa-spotify':'fa-solid fa-music'" class="q-mr-xs" />
```

Comment:
こんな感じで改行したほうが読みやすそうって思ったけどいかがでしょうか...?
```suggestion
        <q-icon :name="roomDetails.provider=='spotify' ? 
          'fa-brands fa-spotify' : 'fa-solid fa-music'"
         class="q-mr-xs"
         />
```

Reply:
どっちでもいいかなー

自分は1行の方で慣れてるから、1行のほうが同じ列に処理がまとまってるから見やすいなーとは思う...

Reply:
じゃあ一旦それでお願いします！！

Reply:
マージしちゃってください🙏